docs/src/_changelog.md

```diff
@@ -6,6 +6,12 @@ CurrentModule = CImGui
 This documents notable changes in CImGui.jl. The format is based on [Keep a
 Changelog](https://keepachangelog.com).
 
+## [v5.1.0] - 2025-02-19
+
+### Changed
```

Comment:
Sorry to be nitpicky but this should be under `Added` instead of `Changed` :sweat_smile: